easyDiffractionApp/Gui/Components/AnalysisDataChartMatplotlib.qml

```diff
@@ -119,11 +121,12 @@ Rectangle {
         }
     }
 
+    /*
     Rectangle {
         id: differenceDataChartContainer
 
-        visible: ExGlobals.Constants.proxy.experimentLoaded
-        height: ExGlobals.Constants.proxy.experimentLoaded ? parent.height * 0.25 : 0
+        visible: ExGlobals.Constants.proxy.showDifferenceChart && ExGlobals.Constants.proxy.experimentLoaded
+        height: visible ? parent.height * 0.25 : 0
```

Comment:
This matters because in this case the `anchors.bottom` of the `analysisDataChartContainer` component is related to the `differenceDataChartContainer.top`. But for components within a column, for example, this check is not needed.